src/DIRAC/Core/Utilities/File.py

```diff
@@ -209,39 +209,6 @@ def getGlobbedFiles(files):
   return globbedFiles
 
 
-def getCommonPath(files):
```

Comment:
I'm 90% sure this is equivalent to `os.path.commonprefix`?

Reply:
How DARE you use standard libraries ?!?! :fearful:

Reply:
How DARE pylint find bugs in the DIRAC implementation that I can't be bothered to fix?! 😄

Reply:
I think pylint should be fixed